### PR TITLE
Tech Insights page

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
## Motivation

We want to show people what our Tech Insights is about

## Approach

Created a landing page with illustration as with the other parts of the product

[Page preview](https://deploy-preview-991--roadie.netlify.app/product/tech-insights/)

### CODE REVIEW

File: src/components/landing/FeatureBlocks/AlternatingFeatureBlock.js [hunk 1]
```diff
@@ -27,6 +28,15 @@ const FeatureBlock = ({
       })}
     >
       <Title el="h3" className="xl:text-2xl xl:tracking-tight text-primary-600">
+        {comingSoon && (
+          <>
+            <span className="inline-block border border-blueroadie text-blueroadie mb-2 text-sm p-2 rounded-lg">
+              Coming soon
```
Comment (jorgelainfiesta) [inline src/components/landing/FeatureBlocks/AlternatingFeatureBlock.js:34]: What's the objective of adding "Coming soon" here? I think it kills the entire page to put it here, I would immediately close the page tbh

File: src/pages/product/tech-insights.js [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
+import classNames from 'classnames';
+
+const SEO_TITLE = 'Tech Insights: scorecards for Backstage [coming soon]';
+const LEAD = `Roadie’s Backstage-based Tech Insights lets you measure and improve software quality`;
+
+const PRODUCT = {
+  features: [
+    {
+      title: 'Raise the bar',
```
Comment (jorgelainfiesta) [inline src/pages/product/tech-insights.js:24]: This is too ambiguous. You need headers to have very concrete value props and how your product will provide it. This title and subtitle could be used to offer coaching or consulting for instance, or for many other kinds of products

File: src/pages/product/tech-insights.js [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+        alt: '',
+      },
+      paragraphs: [
+        'Set expectations around software quality through gamification. Let teams see their quality level vs. the teams around them.',
+        'Gently push teams towards imporovment with prompts in Slack channels and on pull requests.',
+      ],
+    },
+    {
+      title: 'Run migrations with ease',
+      description: 'Roadie gives you insights into library or API migrations',
```
Comment (jorgelainfiesta) [inline src/pages/product/tech-insights.js:63]: I guess nitpick but the rest of the subtitles have an imperative conjugation except this one.

File: src/pages/product/tech-insights.js [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+    {
+      title: 'Completely flexible. Integrates with anything',
+      description:
+        'Define checks against code, third-party APIs, and more...',
+      illustration: {
+        png: IntegrationsImg,
+        alt: '',
+      },
+      paragraphs: [
+        `Create automated checks that integrate with the tools you already use, like PagerDuty, Datadog, and GitHub. Build your own integration if we haven't supplied it.`,
```
Comment (jorgelainfiesta) [inline src/pages/product/tech-insights.js:44]: "Build your own if we haven't supplied it" doesn't sound too optimistic, I'd frame this capability around things that are unique to orgs like internal APIs or on-prem services or something like that instead

File: src/pages/product/tech-insights.js [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+        'Create checks against source code to verify minimum library versions, ensure CI is configured, or require README files to exist.',
+      ],
+    },
+    {
+      title: 'Drive improvements over time',
+      description: 'Nudge teams towards building better software',
```
Comment (jorgelainfiesta) [inline src/pages/product/tech-insights.js:50]: I think it's not great that both the title and subtitle talk about nothing concrete. The subtitle should elaborate on how the value stated in the title is realized through Roadie

**Review by jorgelainfiesta — COMMENTED** [review]
@dtuite I think the approach to shortening the texts is ok, but I think we need to inject more concrete details into the page.

**Review by dtuite — APPROVED** [review]

**Pull request merged**